src/core/include/openvino/op/convert.hpp

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include "openvino/op/op.hpp"
+#include "openvino/op/util/unary_elementwise_arithmetic.hpp"
```

Comment:
As the `Convert` inheritance change was reverted, this header is redundant here.
```suggestion
```